electron/index.d.ts

```diff
@@ -1864,12 +1864,7 @@ declare namespace Electron {
 		/**
 		 * Writes data to the clipboard.
 		 */
-		write(data: {
-			text?: string;
-			rtf?: string;
-			html?: string;
-			image?: NativeImage;
-		}, type?: ClipboardType): void;
+		write(data: { text: string; bookmark?: string; } | { rtf: string; } | { html: string; } | { image: NativeImage; }, type?: ClipboardType): void;
```

Comment:
I don't know why previous `write` definition allows to compile `clipboard.write("test_string")` fine... It should occur an error because first argument is an object.